12_listcomprehension_lab/02.py

```diff
@@ -1,6 +1,10 @@
-"""
-Write a python program that takes two words
-as sys.argv and prints only the letters
-common to both
-"""
-
+import sys 
+import os
+
+a = range (97,123)
+b = range (97,123)
+c = range (97,123)
+
+letters = (range(97,123))
+match = [chr(x)+chr(y)+chr(z) for x in letters for y in letters for z in letters]
+print match
```

Comment:
* a,b,c seem to be unused here.
* There is no need for the parentheses around range in line 8